feature/security/tests/secure_initial_credentials/README.md

```diff
@@ -0,0 +1,18 @@
+# Sec1: Secure Initial Credentials
+
+## Summary
+
+Ensure that secure initial credentials can be set on the router, as well as can prevent any insecure access.
+
+## Procedure
+
+*   Device allows login/password to be set
```

Comment:
The "bootloader password" section is missing. I believe the "login/password" section of the test plan is actually about the console. There is a separate "SSH credentials" section which is what the code currently implements.

I think in the interest to keep test plan small, should we break SEC-2 into several tests?

* SEC-2.1: Secure Initial Bootloader Credentials
* SEC-2.2: Secure Initial Console Credentials
* SEC-2.3: Secure Initial SSH Credentials
* SEC-2.4: Disallow Telnet/Rlogin
* SEC-2.5: Disallow SSH From Data Port

@dplore for feedback.